setup.cfg

```diff
@@ -43,8 +44,6 @@ install_requires=
     keyring >= 15.1
     rfc3986 >= 1.4.0
     colorama >= 0.4.3
-setup_requires =
```

Comment:
I'm about to add this change (removal) in jaraco/skeleton. It does mean dropping support for all but fairly recent setuptools (~2 y.o.).